apps/apps.c

```diff
@@ -238,11 +238,13 @@ int ctx_set_verify_locations(SSL_CTX *ctx, const char *CAfile,
 int ctx_set_ctlog_list_file(SSL_CTX *ctx, const char *path)
 {
     if (path == NULL) {
-        if (SSL_CTX_set_default_ctlog_list_file(ctx) <= 0) {
-            BIO_puts(bio_err, "Failed to load default Certificate Transparency "
-                     "log list\n");
-        }
-        return 1; /* Do not treat failure to load the default as an error */
+        /*
+         * Try and load the log list from the default location.
+         * We don't treat it as an error if this fails, since the user didn't
+         * explicitly request a log list be loaded and might not need it either.
+         */
+        SSL_CTX_set_default_ctlog_list_file(ctx);
```

Comment:
Hm, new boolean functions should generally be __owur (stands for OpenSSL warn unused result, see the header for examples). It's better to make it an error in general, and swallow the error here. Alternatively, you could just downgrade the message to "Info: no default Certificate Transparency log list available."

Also - not for the scope of this PR, but out of curiosity, when does this return '<0'? I think (though that's my personal preference) that we should keep new APIs 1/0 whenever possible.